content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -466,43 +466,38 @@ To set a different service name per Ingress using annotations:
 ```
 The above overrides the default `nginx-ingress-controller.ingress-nginx` service name.
 
-### Sampling
+### Ingress Controller Sampling
 
-To control the volume of traces starting from Nginx that are sent to Datadog, specify a sampling rate by setting the parameter `DD_TRACE_SAMPLING_RULES` to a value between `0.0` (0%) and `1.0` (100%). If no value is specified, 100% of traces starting from Nginx are sent. The Kubernetes Nginx ingress controller uses [v1.2.1][11] of the `dd-opentracing-cpp` library).
+To control the volume of Ingress Controller traces that are sent to Datadog, specify a sampling rule that matches all traces. The `sample_rate` configured in the rule will determine the proportion of traces that are sampled.
```

Comment:
Should we call out not to use `datadog-sample-rate`? https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#datadog-sample-rate

Reply:
Possibly, but I think that it's better to take the risk and not mention it.

When the most recent release of the Datadog tracer is released in the Ingress Controller (probably within the next few weeks), then `datadog-sample-rate` will work again.

By not mentioning it, these instructions remain valid for both the "broken" and "fixed" releases.

Reply:
Thanks for clarifying!